packages/docs/celo-codebase/protocol/plumo.md

```diff
@@ -71,25 +71,28 @@ The instructions below are for the first phase of the Plumo setup, Powers of Tau
 
 ### Generate your address
 
-The first step to participating is generating your Celo address.
+The first step to participating is generating your Celo address in a secure location.
 
 * Download the `generate` binary corresponding to your OS from [here](https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0-benchmarking.4).
 * Run it in a command line - navigate in the command line to the relevant folder - and follow the instructions.
     * When asked to `Enter some entropy for your Plumo seed:`, you can use any source of entropy.
     * Make sure to save your passphrase - you will need it later.
-* Send the address generated to kobi@clabs.co. Keep the resulting `plumo.keys` file.
+* Send the address generated to kobi@clabs.co.
+* Keep the resulting `plumo.keys` file. Ideally, it will be stored in a location that's easy to destroy afterwards, such as a USB thumbdrive.
 
 ### Run the Contributor software
 
 Next you'll download the `contribute` binary and begin contributing to the Plumo setup. 
 
 cLabs is running the coordinator server, which has a list of approved participant addresses. Additionally, cLabs is running a few verifiers that verify contributions on-demand, allowing the setup to make progress.
 
-* After receiving confirmation from cLabs, download the `contribute` binary from [here]https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0-rc.1).
-* Run it in the same directory as the `plumo.keys` file as follows:
-    * Windows users: `contribute-windows --coordinator-url https://plumo-setup-phase-1.azurefd.net`
-    * macOS users: `./contribute-macos --coordinator-url https://plumo-setup-phase-1.azurefd.net`
-    * Linux users: `./contribute-linux --coordinator-url https://plumo-setup-phase-1.azurefd.net`
+* After receiving confirmation from cLabs, download the `contribute` binary from [here]https://github.com/celo-org/snark-setup-operator/releases/tag/v1.0.0).
+* Run it as follows, assuming your keys reside in KEYS_PATH:
+    * Windows users: `./contribute-windows.exe --keys-path KEYS_PATH`
+    * macOS users: `./contribute-macos --keys-path KEYS_PATH`
+    * Linux users: `./contribute-linux --keys-path KEYS_PATH`
```

Comment:
```suggestion
* Run it as follows, replacing the env var $KEYS_PATH with the location of your keys:
    * Windows users: `./contribute-windows.exe --keys-path $KEYS_PATH`
    * macOS users: `./contribute-macos --keys-path $KEYS_PATH`
    * Linux users: `./contribute-linux --keys-path $KEYS_PATH`
```

Reply:
Not sure about this. Are env vars convenient for non-developers?

Reply:
We could include a line that helps them set it eg.
`export $KEYS_PATH = "/mnt/plumo/plumo.keys"`

But no worries, can skip if more confusing